src/main/kotlin/me/partlysanestudios/partlysaneskies/features/chat/ChatAlertsManager.kt

```diff
@@ -0,0 +1,203 @@
+    fun load() {
+        val file = File(DATA_PATH_NAME)
+
+        if (file.createNewFile()) {
+            file.setWritable(true)
+            FileWriter(file).use { writer ->
+                writer.write(Gson().toJson(ArrayList<String>()))
+            }
+        }
+
+        file.setWritable(true)
+        Files.newBufferedReader(Paths.get(file.path)).use { reader ->
+            chatAlertsList = Gson().fromJson(reader, ArrayList::class.java) as ArrayList<String>
+        }
+    }
+
+    fun registerCommand() {
+        PSSCommand("chatalerts")
+            .addAlias("ca", "chatAlert", "chal")
+            .setDescription("Operates the chat alerts feature: /chatalerts <add/remove/list> ")
+            .setRunnable { args ->
+                when {
+                    args.isEmpty() -> ChatUtils.sendClientMessage("§cIncorrect usage. Correct usage: /chatalerts add/remove/list")
+                    else -> when (args[0]) {
+                        "list" -> listAlerts()
+                        "add" -> {
+                            if (args.size == 1) {
+                                ChatUtils.sendClientMessage("§cIncorrect usage. Correct usage: /chatalerts add [alert]")
+                            } else {
+                                val alert = args.copyOfRange(1, args.size).joinToString(" ").trim()
+                                addAlert(alert)
+                            }
+                        }
+                        "remove" -> {
+                            if (args.size == 1) {
+                                ChatUtils.sendClientMessage("§cIncorrect usage. Correct usage: /chatalerts remove [number]")
+                            } else {
+                                args[1].toIntOrNull()?.let { removeAlert(it) } ?: run {
+                                    ChatUtils.sendClientMessage("§c\"${args[1]}\" could not be read as a number. Correct Usage: /chatalerts remove [number]")
+                                }
+                            }
+                        }
+                        else -> ChatUtils.sendClientMessage("§cIncorrect usage. Correct usage: /chatalerts add/remove/list")
+                    }
+                }
+            }.register()
+    }
+
+    @Throws(IOException::class)
+    fun save() {
+        val file = File(DATA_PATH_NAME)
+        val gson = GsonBuilder().setPrettyPrinting().serializeSpecialFloatingPointValues().create()
+        val json = gson.toJson(chatAlertsList)
+        FileWriter(file).use { writer ->
+            writer.write(json)
+        }
+    }
+
+    private fun addAlert(alert: String) {
+        chatAlertsList += alert
+        try {
+            save()
+        } catch (e: IOException) {
+            ChatUtils.sendClientMessage("§cChat Alerts was unable to save. Please try again.")
+            e.printStackTrace()
+        }
+        ChatUtils.sendClientMessage("§b\"§d$alert§b\" was successfully added as alert number §d${chatAlertsList.size}§b.")
```

Comment:
Doesn't this emply the following possible message combination?

"Chat Alerts was unable to save. Please try again.
(alert) was succesfully added as alert number (number)"

Reply:
very true will fix